<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionsWidget</class>
 <widget class="QDockWidget" name="FunctionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowTitle">
   <string notr="true">Functions</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 0px;
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="functionsTreeView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeView::item
{
    padding-left:10px;
    padding-top: 1px;
    padding-bottom: 1px;
    border-left: 10px;
}

QTreeView::item:hover
{
    background: rgb(242, 246, 248);
    color: black;
}

QTreeView::item:selected
{
    background: gray;
    color: white;
}


QToolTip
{
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
	font: 9pt &quot;Noto&quot;;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tree</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="nestedFunctionsTree">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeWidget::item
{
    padding-left:10px;
    padding-top: 1px;
    padding-bottom: 1px;
    border-left: 10px;
}

QTreeWidget::item:selected
{
    background: gray;
    color: white;
}

QTreeWidget::branch:selected
{
    background: gray;
}
QTreeWidget::item:hover
{
    background: rgb(242, 246, 248);
    color: black;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
	font: 11pt &quot;Monaco&quot;;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionDisasAdd_comment">
   <property name="text">
    <string>Add comment</string>
   </property>
  </action>
  <action name="actionFunctionsRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionFunctionsUndefine">
   <property name="text">
    <string>Undefine</string>
   </property>
  </action>
  <action name="action_References">
   <property name="text">
    <string>X-Refs</string>
   </property>
   <property name="toolTip">
    <string>Cross references</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
